Housekeeping

huayunh opened the pull request:
#### Changes proposed in this Pull Request:

- Changed links to point to GHE
- Changed code owners
- Added disclaimers for React Native Paper v5
- Added contributing guidelines & code of conduct
  - Note that these links wont work until we merge these diffs to master

huayunh commented:
@joebochill I heard that there were some discussions about commenting out the CI step involving link checks. I have found that step to be helpful, as there have been numerous cases where CI caught some bona fide 404 links.

joebochill commented:
> @joebochill I heard that there were some discussions about commenting out the CI step involving link checks. I have found that step to be helpful, as there have been numerous cases where CI caught some bona fide 404 links.

If it is working as expected, I don't have an issue with it. I know we've had a lot of problems with it flagging valid links — have these issues been resolved (in a way that doesn't just ignore those links)?

huayunh commented:
@joebochill 
- We replaced 404 links and 300 links whenever we found them.
- All remaining ignored links are currently skipped because their HTTP status code is returning 403, and I interpret that as "we don't want to be link checked by your script". 
- All MDX 2-related issues should have gone away by now; we changed the comment syntax to be like `{/* <!-- xxx --> */}`

joebochill commented:
> @joebochill
> 
> * We replaced 404 links and 300 links whenever we found them.
> * All remaining ignored links are currently skipped because their HTTP status code is returning 403, and I interpret that as "we don't want to be link checked by your script".
> * All MDX 2-related issues should have gone away by now; we changed the comment syntax to be like `{/* <!-- xxx --> */}`

Are the 403 issues coming just from github links, or is it other domains too? I read something about a workaround for github links, but either way if we're happy with how it's working for the moment, i'm cool with that.

joebochill commented:
Looks like the only broken link now is the self-referential link to contributing guide which won't exist until this PR is merged...we can throw an ignore comment around that to get this to pass.
